test/react-native/features/fixtures/app/App.js

```diff
@@ -1,11 +1,16 @@
 import React, { useContext, useEffect } from 'react'
-import { SafeAreaView, StyleSheet, View, Text, RootTagContext } from 'react-native'
+import { SafeAreaView, StyleSheet, View, Text, RootTagContext, unstable_RootTagContext } from 'react-native'
 import { launchScenario } from '@bugsnag/react-native-performance-scenarios'
 
 console.reportErrorsAsExceptions = false
 
 const App = () => {
-  const rootTag = useContext(RootTagContext)
+  let rootTag
+  if (RootTagContext) {
+    rootTag = useContext(RootTagContext)
+  } else if (unstable_RootTagContext) {
+    rootTag = useContext(unstable_RootTagContext)
+  }
```

Comment:
would this not work with `rootTag = useContext(RootTagContext || unstable_RootTagContext)` ?